styles/tree-view.less

```diff
@@ -73,7 +73,7 @@
   width: 2px;
   background: @base-accent-color;
   opacity: 0;
-  transform: scaleY(0);
+  // transform: scaleY(0);
```

Comment:
You can delete the whole line. Also, the `, transform .16s cubic-bezier(.80, 0, .90, .53)` part from the transition below.